panel/widgets/button.py

```diff
@@ -171,7 +171,7 @@ class Button(_ClickButton):
 
     :Example:
 
-    >>> pn.widgets.Button(name='Click me', button_type='primary')
+    >>> pn.widgets.Button(name='Click me', icon='caret-right', button_type='primary')
```

Comment:
Will help our users discover the new `icon` feature. Using this often will help our users create apps with a good UX.